lib/example/example.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Plexy.Example do
+  use Plug.Router
+  plug Plexy.RequestId
+  plug Plexy.Instrumentor
```

Comment:
Honestly, I like being able to pick and choose here. Pliny does the same where you the user is in control of which middleware and which order.